plugin/outline.vim

```diff
@@ -64,19 +64,19 @@ import autoload "../lib/outline.vim"
 # \ :call <SID>outline.Toggle()<cr>
 noremap <unique> <script> <Plug>OutlineToggle
             \ :call <SID>outline.Toggle()<cr>
-if !hasmapto("<Plug>OutlineToggle" ) || empty(mapcheck("<F8>", "n"))
+if hasmapto("<Plug>OutlineToggle" ) && empty(mapcheck("<F8>", "n"))
```

Comment:
I tested, it should be:
`if !hasmapto("<Plug>OutlineToggle" ) || empty(mapcheck("<F8>", "n"))` 

The same applies for the other mappings. :)